tests/config/state/tests.yml

```diff
@@ -39,6 +39,9 @@ components:
   - component: cassandra
     allOperations: false
     operations: [ "set", "get", "delete", "bulkset", "bulkdelete", "ttl" ]
+  - component: cloudflare.kv
+    allOperations: false
+    operations: [ "set", "get", "delete", "bulkset", "bulkdelete"]
```

Comment:
As the basis is a key value store primarily, should we support timeToLive since beginning?

Reply:
Right, looks like TTLs should be supported: https://developers.cloudflare.com/workers/runtime-apis/kv/#expiring-keys I need to add support for them

Reply:
I added support for TTLs. However, it seems that KV requires TTLs to be > 60 seconds. For that reason, I am not enabling them on conf tests. It should be tested in cert tests in the future.